ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/point/DropAPinMapFragment.kt

```diff
@@ -99,19 +99,19 @@ class DropAPinMapFragment(private val viewModel: DropAPinTaskViewModel) :
     }
   }
 
-  override fun onMapReady(map: Map) {
+  override fun onMapReady(map: MapView) {
     viewModel.features.observe(this) { map.renderFeatures(it) }
   }
 
   override fun getMapViewModel(): BaseMapViewModel = mapViewModel
 
-  override fun onMapCameraMoved(position: CameraPosition) {
-    super.onMapCameraMoved(position)
+  fun onMapCameraMoved(position: CameraPosition) {
+    super.getMapViewModel().onMapCameraMoved(position)
```

Comment:
I believe this won't work. Earlier we were overriding the `onMapCameraMoved` which was getting called when the map moved. But now, it is just a method with no callers.

Also, the `super` keyword can be omitted too.